api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -1344,7 +1344,29 @@ public RecordingResponse listRecordings(String callID, String legId, final Integ
 
         return messageBirdService.requestList(url, offset, limit, RecordingResponse.class);
     }
+    /**
+     * Deletes a voice recording
+     *
+     * @param callID Voice call ID
+     * @param legID Leg ID
+     * @param recordingID
+     * @throws UnauthorizedException if client is unauthorized
+     * @throws GeneralException      general exception
+    */
+    public void deleteRecording(final String callID, final String legID, final String recordingID)
+            throws NotFoundException, GeneralException, UnauthorizedException {
```

Comment:
null check and throw exception for call id leg id and recording id as below;
if (callID == null) {
    throw new IllegalArgumentException("Call ID must be specified.");
}

Reply:
I've added an example and the requested null checks.